spec/factories/schools.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
-  OFSTED_RATINGS = ['Outstanding', 'Good', 'Requires Improvement', 'Inadequate']
-  RELIGIOUS_CHARACTERS = ['Church of England', 'Roman Catholic', 'None', 'Does not apply']
+  OFSTED_RATINGS = ['Outstanding', 'Good', 'Requires Improvement', 'Inadequate'].freeze
```

Comment:
`%w[]`?

Reply:
you cannot do it when one value has a space in it
`Requires Improvement`